src/main/java/core/mate/academy/service/impl/BulldozerProducer.java

```diff
@@ -0,0 +1,19 @@
+package core.mate.academy.service.impl;
+
+import core.mate.academy.model.Bulldozer;
+import core.mate.academy.model.Machine;
+import core.mate.academy.service.MachineProducer;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer {
+    private List<Bulldozer> bulldozerList = new ArrayList<>();
+
+    @Override
+    public List<? extends Machine> get() {
```

Comment:
This service must only produce Bulldozers. Let's parametrize class MachineProducer with Bulldozer, like this
`MachineProducer<Bulldozer>`